README.md

```diff
@@ -75,4 +89,20 @@ Africa/Algiers
 `worldtimebuddy --timezone UTC`
 
 UTC                            2024-10-06 13:52:37
+...
+
+
+6. Getting a difference of time from a particular timezone 
+(supported time formats: +/- min, hr, day)
```

Comment:
Put a line break here.

Reply:
the whole usage section is in a single code block, should I fix it and put individual sample code in code blocks?

Reply:
No just put an extra line after line#96

Reply:
> the whole usage section is in a single code block, should I fix it and put individual sample code in code blocks?

This would be great! If you can do it?